test/core_fusion.cpp

```diff
@@ -648,6 +650,70 @@ TEST(core_fusion, DISABLED_conv_bias_bprop)
     }
 }
 
+#ifndef NGRAPH_JSON_DISABLE
+TEST(batch_fusion, fuse_batch_mat_mul_transpose)
```

Comment:
Should these tests be moved to some file for all backends to use since the op is not an ngraph fused op

Reply:
right, i will move to tests/backend/batch_mat_mul.in.cpp

Reply:
Done.
moved the other 2 tests which need a backend to backend/batch_matmul.in.cpp 
this particular test `fuse_batch_mat_mul_transpose` is kept here as it is not related to any backend.